client/shared-settings/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { combineReducers } from 'redux';
+import SharedSettingsRootView from './views';
+import reducer from 'lib/form-base/reducer';
+// from calypso
+import notices from 'state/notices/reducer';
+
+export default ( { formData, formMeta, storeOptions } ) => ( {
+	getReducer() {
+		return combineReducers( {
+			form: reducer,
+			notices,
+		} );
+	},
+
+	getHotReducer() {
+		return combineReducers( {
+			form: reducer,
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/488/commits/8ebe257dc446a63b65a6a909757e8a51ee1eb35a